AdyenCard/Utilities/CardType/CardTypeDetector.swift

```diff
@@ -13,7 +13,7 @@ extension Array: AdyenCompatible {}
 
 /// Adds helper functionality to any `[CardType]` instance through the `adyen` property.
 @_spi(AdyenInternal)
-public extension AdyenScope where Base == [CardType] {
```

Comment:
package. please remember our goal, we should avoid public access where possible

Reply:
yes, I remember but I made this to keep changes minimum as discussed. If I change here then I might have to change access specifier in other places as well which I did not want.

Reply:
`package` here should work, as it's a new change, we are not using any spi code under here.